Jenkinsfile

```diff
@@ -111,6 +145,29 @@ pipeline {
                 }
             }
         }
+        stage('Integration test') {
+            steps {
+                script {
+                    if (isTimeTriggeredBuild() && (branchName == 'development' || env.CHANGE_ID)) {
+                        container('theia') {
+                            dir('tests') {
+                                sh './theiaPrepare.sh'
+                            }
+                        }
+                        container('python') {
+                            dir('tests/theia_automation_lsp') {
+                                sh 'pip install -r requirements.txt'
+                                sh 'apt update'
+                                sh 'apt install firefox-esr -y'
+                                sh 'PYTHONPATH=`pwd` robot -i Rally -e Unstable --variable HEADLESS:True --outputdir robot_output robot_suites/lsp/local/firefox_lsp_local.robot'
```

Comment:
is it possible to have it also for chrome? or this is only first portion, and chrome will follow.

Reply:
The tests are the same. As I understand, there is no need to test on both browsers.
I also faced issues trying to run tests on chrome and choose firefox because it is a more straightforward solution.
I'm not plan to add chrome support in this request. But it could be done by a separate change, if needed.